### ISSUE TITLE
Use system background color in window mode

### ISSUE THREAD

**Issue by macias** [BestImageViewer/geeqie#926]
* Feature Request

```
Geeqie 1.6
```

```
What GTK toolkit is used to compile Geeqie? I have no idea, it comes as a package from openSUSE repo.
```
openSUSE Tumbleweed.

### SUMMARY

Please provide system background color as default in window mode.

### STEPS TO REPRODUCE

* Set the theme of your desktop environment to some kind of bright.
* Run geeqie. Observe it uses black background around the image in window mode instead of bright one (from the theme).
* It would be much more smooth experience if geeqie would use instead of black color the system color. Black of course is a good match if someone uses theme like dark-high-contrast.

**Comment by caclark** [comment]
The background colour can be set on `Edit/Preferences/Image/Border Colour`

For people who regularly switch themes (e.g. day/night) it would be useful to automatically change the background colour.
It is possible to do so, but as it is not so straightforward it is unlikely to happen in the near future.

**Comment by macias** [comment]
If this is the only way, then I would rephrase this to having two separate settings -- for full screen screen and window mode. Because if you change one, it affects the other mode (since there is only one color setting for it).
